insights/parsers/satellite_postgresql_query.py

```diff
@@ -72,15 +72,24 @@ class SatellitePostgreSQLQuery(CommandParser, list):
 
     Raises:
         SkipException: when there isn't data in the table
-        ParseException: when the output isn't in good csv format
+        ParseException: when the output isn't in good csv format or the subclass doesn't define headers
     """
+    headers = []
 
     def parse_content(self, content):
-        if not content or len(content) == 1:
-            raise SkipException("There is no data in the table")
+        if self.__class__.__name__ != 'SatellitePostgreSQLQuery' and not self.headers:
```

Comment:
When defining the base class like this (use an undefined/empty static member as a required `gurad`), it's not a good idea to instantiate or use it like this. that's to say, this base class should be instantiated due the `self.headers` is never contain a value, unless in its child classes. 

Better remove all the direct tests against the base class. but test them in the child-classes instead.

Reply:
@xiangce I update the code to move the test in child-classes, please help to review again, thanks.